src/views/InternalApp/ThePlans/components/CreatePlan.vue

```diff
@@ -50,6 +58,7 @@ onMounted(() => {
   <PlanForm 
     :file="imageFile"
     :formType="formType"
+    :is-file-valid="isFileValid"
```

Comment:
Is this the format on props? Please review the docs
https://app.clickup.com/14221792/v/dc/dj0f0-2765/dj0f0-225